src/mgr/ActivePyModules.cc

```diff
@@ -434,7 +436,16 @@ PyObject *ActivePyModules::get_python(const std::string &what)
       mgr_map.dump(&f);
     });
     return f.get();
-  } else {
+  } else if (what == "osd_df"){
+    cluster_state.with_osdmap_and_pgmap([&f, &what, &tstate](const OSDMap& osd_map, const PGMap& pg_map){
+      string class_name{""};
+      string item_name{""};
+      PyEval_RestoreThread(tstate);
+      OSDUtilizationFormatDumper d(crush, &osd_map, pg_map, false, class_name, item_name);
```

Comment:
i don't think this compiles. the declaration of `OSDUtilizationFormatDumper` is:

```c++
  OSDUtilizationFormatDumper(const CrushWrapper *crush, const OSDMap *osdmap,
                             const PGMap& pgmap, bool tree,
                             const string& filter) :
```

Reply:
> i don't think this compiles. the declaration of `OSDUtilizationFormatDumper` is:
> 
> ```c++
>   OSDUtilizationFormatDumper(const CrushWrapper *crush, const OSDMap *osdmap,
>                              const PGMap& pgmap, bool tree,
>                              const string& filter) :
> ```

when we compiles this code, the code was not updated yet and i will correct my code later.
'''
OSDUtilizationFormatDumper(const CrushWrapper *crush, const OSDMap *osdmap,
                             const PGMap& pgmap, bool tree,
                             const string& class_name,
                             const string& item_name) :
'''